contracts/src/v0.8/dev/ocr2dr/OCR2DR.sol

```diff
@@ -0,0 +1,249 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
```

Comment:
The folder that this lives in relates to the Solidity version, therefore this should be upgraded to at least:
```suggestion
pragma solidity ^0.8.0;
```

Reply:
OMG, now I see the purpose of the subfolders. I thought this is our internal versioning.. 
Will change accordingly. 
I cannot make it 0.8.0 because it uses CBOR/Buffer for 0.7.0 and AFAIK we cannot mix the versions.
Therefore I will need to move my code under 0.7.0

Reply:
Fixed